sql-gremlin/src/main/resources/error-messages_en.properties

```
@@ -77,3 +77,4 @@ OFFSET_NOT_SUPPORTED=Unsupported: OFFSET is not currently supported.
 UNSUPPORTED_LITERAL_EXPRESSION="Unsupported: Raw literal expressions without any tables referenced are now supported at this time."
 CANNOT_JOIN_DIFFERENT_EDGES="Error: Cannot perform JOIN on two edges that are not the same (%s != %s)."
 UNSUPPORTED_OPERAND_TYPE="Unsupported operand type %s, cannot rename column."
+UNRECOGNIZED_TYPE="Error, unrecognized type: '%s'."
```

Nit: missing newline
```suggestion
UNRECOGNIZED_TYPE="Error, unrecognized type: '%s'."

```